content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -899,6 +899,61 @@ enum Role {
 </tab>
 </TabbedContent>
 
+## Defining composite types
+
+<Admonition type="info">
+
+Composite types are available in Preview in versions `3.10.0` and later.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Composite types are currently only available on MongoDB.
+
+</Admonition>
+
+Composite types (known as [embedded documents](https://docs.mongodb.com/manual/core/data-model-design/#std-label-data-modeling-embedding) in MongoDB) provide support for embedding records inside other records, by allowing you to define new object types. Composite types are structured and typed in a similar way to models.
```

Comment:
I think we never use the term "record" this way.

Also:
```suggestion
Composite types (known as [embedded documents](https://docs.mongodb.com/manual/core/data-model-design/#std-label-data-modeling-embedding) in MongoDB) provide support for embedding records inside other records, by allowing you to define new object types. Composite types are structured and typed in a similar way to [models](#defining-models).
```

Reply:
You mean: "embedding records inside other records"?

Any reason not to? Confusing?

Reply:
Yes I'm also unclear, I though this was how we are using the term 'records'? What's the difference here? https://www.prisma.io/docs/about/prisma-docs/style-guide#record

Reply:
I was not aware of that. All good then.

The suggestion for the link would still be useful I feel.

Reply:
Yep, thanks, I've added the link in